sunpy/conftest.py

```diff
@@ -57,12 +57,14 @@ def pytest_runtest_setup(item):
 
 
 def pytest_unconfigure(config):
+    # If at least one figure test has been run, print result image directory
     if len(new_hash_library) > 0:
         # Write the new hash library in JSON
         figure_base_dir = os.path.abspath(config.getoption("--figure_dir"))
         hashfile = os.path.join(figure_base_dir, HASH_LIBRARY_NAME)
         with open(hashfile, 'w') as outfile:
             json.dump(new_hash_library, outfile, sort_keys=True, indent=4, separators=(',', ': '))
 
+        generate_figure_webpage()
```

Comment:
Do we only want to generate the webpage if there is a difference?

Reply:
Don't see why not, it's a small overhead and just generates a small `.html` file.